libpod/networking_linux.go

```diff
@@ -238,7 +243,8 @@ func (r *Runtime) setupRootlessNetNS(ctr *Container) (err error) {
 					continue
 				}
 				if status.Exited() {
-					return errors.New("slirp4netns failed")
+					slurp, _ := ioutil.ReadAll(stderr)
```

Comment:
I don't think this works. The pipe is auto-closed when Go detects the process has exited.